app/assets/stylesheets/components/header.scss

```diff
@@ -137,29 +139,39 @@
 
 .skip-content-link {
   position: absolute;
-  top: 0;
   left: 50%;
-  transform: translate(-50%, -100px);
   background: var(--body-color);
   color: var(--body-color-inverted);
   z-index: 9999;
   padding: var(--su-4);
   pointer-events: none;
   opacity: 0;
   border-radius: var(--radius);
-  outline: 0;
+  height: var(-header-height);
+  transform: translateY(-10em);
+  transition: transform 0.2s ease-in-out;
 
-  &:focus {
+  &:focus,
+  &:focus-within {
     pointer-events: auto;
     opacity: 1;
-    transform: translate(-50%, var(--header-height) / 2);
+    transform: translateY(0em);
```

Comment:
refactor (non-blocking): Since it's `0em`, `0` can be used instead.

Reply:
@nickytonline That's an oops on my end. Thanks for catching that.